### PR TITLE
Fixes issue where fluid response is treated as empty multi-size response.

### PR DESCRIPTION
**Pull request by glevitzky**
For requests that are both multi-size and fluid, the response may be fluid but may contain a multi-size header indicating the size of the creative. Since fluid creatives by default have no size (i.e., return a size header of '0x0'), this causes the slot to collapse when it should not.

### CODE REVIEW

**Comment by jeffkaufman** [comment]
Is this only supposed to change the behavior when the header is indicating both zero width and zero height?  (It's not clear to me from the PR description)

**Comment by glevitzky** [comment]
Yes, this will only matter when both dimensions are 0. Updated description to reflect that.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -981,7 +981,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     // We want to resize only if neither returned dimension is larger than its
     // primary counterpart, and if at least one of the returned dimensions
     // differ from its primary counterpart.
-    if (this.isFluid_ ||
+    if ((this.isFluid_ && width > 0 && height > 0) ||
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Can just be && width && height assuming you'll never get a negative value

Comment (glevitzky) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Done.

**Pull request merged**